MediaRecorder.bs

```diff
@@ -209,7 +209,7 @@ interface MediaRecorder : EventTarget {
 
       <li>If at any point the {{MediaRecorder/stream}}'s <a>isolation properties</a>
       change so that {{MediaRecorder}} is no longer allowed access to it, the
-      UA MUST immediately stop gathering data, discard any data that it has
+      UA MUST stop gathering data, discard any data that it has
```

Comment:
The changes do not correspond with the title of the PR: 
  "Do not discard data in case track set change"

Reply:
This one is due to a change in the stream's isolation properties, not its track set.
Some backstory on "immediately" in [issue 151 ](https://github.com/w3c/mediacapture-record/issues/151#issuecomment-457446229).

Reply:
@youennf Note, the specification does not explicitly prohibit _replacing_ the track, see 

> [4.8.12.10 Media resources with multiple media tracks](https://html.spec.whatwg.org/multipage/media.html#media-resources-with-multiple-media-tracks)
> 
> A media resource can have multiple embedded audio and video tracks. For example, in addition to the primary video and audio tracks, a media resource could have foreign-language dubbed dialogues, director's commentaries, audio descriptions, alternative angles, or sign-language overlays.
> 
> There are only ever one AudioTrackList object and one VideoTrackList object per media element, even if another media resource is loaded into the element: the objects are reused. (The AudioTrack and VideoTrack objects are not, though.)

https://github.com/w3c/mediacapture-record/issues/147#issuecomment-480580164

Reply:
@Pehrsons Related https://bugzilla.mozilla.org/show_bug.cgi?id=1557394